front_matter/any.ts

```diff
@@ -3,9 +3,15 @@
 import { createExtractor, type Parser } from "./_create_extractor.ts";
 import { parse as parseYaml } from "@std/yaml/parse";
 import { parse as parseToml } from "@std/toml/parse";
-import type { Extractor } from "./types.ts";
+import type { Extract, Extractor } from "./types.ts";
```

Comment:
There no longer seems a reason to export `Extractor`.
```suggestion
import type { Extract } from "./types.ts";
```